atest/interpreter.py

```diff
@@ -48,6 +49,23 @@ def _get_name_and_version(self):
         version = re.match(r'\d+\.\d+\.\d+', version).group()
         return name, version
 
+    def _get_java_version(self):
+        if not self.is_jython:
+            return -1, -1
+        try:
+            # platform.java_ver() returns Java version in a format:
+            # ('9.0.7.1', 'Azul Systems, Inc.', ('OpenJDK 64-Bit Server VM', '9.0.7.1+1', 'Azul Systems, Inc.'), ('Windows 10', '10.0', 'amd64'))
+            # ('11.0.6', 'Ubuntu', ('OpenJDK 64-Bit Server VM', '11.0.6+10-post-Ubuntu-1ubuntu118.04.1', 'Ubuntu'), ('Linux', '4.4.0-18362-Microsoft', 'amd64'))
+            # ('1.8.0_121', 'Oracle Corporation', ('Java HotSpot(TM) 64-Bit Server VM', '25.121-b13', 'Oracle Corporation'), ('Windows 10', '10.0', 'amd64'))
```

Comment:
I don't think this comment is needed. At least it doesn't need to be this verbose but I'd say those interested in `platform.java_ver()` return value can check the docs.